odoo/addons/test_lint/tests/lint_case.py

```diff
@@ -16,7 +16,23 @@ def iter_module_files(self, *globs):
         """
         for modroot in map(get_module_path, get_modules()):
             for root, _, fnames in os.walk(modroot):
-                fnames = [j(root, n) for n in fnames]
+                fnames = [os.path.join(root, n) for n in fnames]
                 for glob in globs:
                     fnames = fnmatch.filter(fnames, glob)
                 yield from fnames
+
+
+class NodeVisitor(ast.NodeVisitor):
```

Comment:
Don't inherit from `ast.NodeVisitor` at all since it's not used, and document the purpose of this alternate visitor.